templates/speech_recognition/ASR/train.py

```diff
@@ -471,7 +472,7 @@ def text_pipeline(words):
     # you can train from scratch and avoid this step.
     # We download the pretrained LM from HuggingFace (or elsewhere depending on
     # the path given in the YAML file). The tokenizer is loaded at the same time.
-    hparams["pretrainer"].collect_files()
+    hparams["pretrainer"].collect_files(local_strategy=LocalStrategy.SYMLINK)
```

Comment:
Now unneeded, as this is the default again.

```suggestion
    hparams["pretrainer"].collect_files()
```